x-pack/test/security_functional/unsecure_cluster_alert.config.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { resolve } from 'path';
+import { FtrConfigProviderContext } from '@kbn/test';
+import { services } from '../functional/services';
+import { pageObjects } from '../functional/page_objects';
+
+// the default export of config files must be a config provider
+// that returns an object with the projects config values
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const kibanaCommonConfig = await readConfigFile(
+    require.resolve('../../../test/common/config.js')
+  );
+  const kibanaFunctionalConfig = await readConfigFile(
+    require.resolve('../../../test/functional/config.base.js')
+  );
+
+  const kbnTestServerArgOverrides = [
+    ...kibanaCommonConfig
+      .get('kbnTestServer.serverArgs')
+      .filter((arg: string) => !arg.startsWith('--security.showInsecureClusterWarning')),
+  ];
+
+  kbnTestServerArgOverrides.push('--security.showInsecureClusterWarning=true');
```

Comment:
__nit:__ I believe this value is `true` by default and we don't need explicitly set it here?